tests/components/wemo/conftest.py

```diff
@@ -40,6 +40,14 @@ def on_func(device, type_filter, callback):
         yield registry
 
 
+@pytest.fixture(name="pywemo_discovery_responder", autouse=True)
+def pywemo_discovery_responder_fixture():
+    """Fixture for the DiscoveryResponder instance."""
+    discovery_responder = create_autospec(pywemo.ssdp.DiscoveryResponder, instance=True)
+    with patch("pywemo.ssdp.DiscoveryResponder", return_value=discovery_responder):
```

Comment:
What's the reason we don't just pass the autospec parameter to patch?

Reply:
I do it this way to avoid needing to handle multiple mock instances of the patched class. That said, since no code is using the responder mock instance, there is no benefit to using a separate create_autospec here.

Updated the code to use `autospec=True` and a bare `yield`.